lib/pages/chat/chat.dart

```diff
@@ -1469,8 +1497,21 @@ class ChatController extends State<Chat>
     }
   }
 
-  void downloadFileAction(BuildContext context, Event event) =>
-      event.saveFile(context);
+  Future<String?> downloadFileAction(BuildContext context, Event event) async =>
+      await event.saveFile(context);
+
+  void previewPdfWeb(BuildContext context, Event event) async {
+    final pdf = await event.getFile(context);
+    if (pdf.result == null || event.sizeString != pdf.result?.sizeString) {
+      Fluttertoast.showToast(msg: L10n.of(context)!.errorGettingPdf);
+      return;
+    }
+
+    final blob = html.Blob([pdf.result!.bytes], 'application/pdf');
+    final url = html.Url.createObjectUrlFromBlob(blob);
+    html.window.open(url, "_blank");
+    html.Url.revokeObjectUrl(url);
```

Comment:
What do this do ?

Reply:
releases the url created before after using it because it's useless afterward

https://developer.mozilla.org/fr/docs/Web/API/URL/revokeObjectURL_static